packages/react-workspace/react-client-app/src/components/App.tsx

```diff
@@ -97,6 +597,7 @@ export function App(): JSX.Element {
                     <div className="sub-container">
                         <div className="container-label">Anchor</div>
                         <div><NimbleAnchor href="#" appearance="prominent">Site root</NimbleAnchor></div>
+                        Let's try it <NimbleAnchor href="#" underline-hidden={false}>using href</NimbleAnchor>.
```

Comment:
nit: looks like the Angular example app has slightly different configuration for this link. Let's try to keep them in sync across frameworks.

https://github.com/ni/nimble/blob/4e353887ac1607dd2a05c0544e2b6189dd90c03c/packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.html#L12

Reply:
routerLinks only apply to angular, we can follow blazor and just not have this in the example